src/aws/auth.service.ts

```diff
@@ -30,14 +30,18 @@ export class AuthService {
       // This is so that the role-based access is determined by Cognito groups
       const groups = userPayload["cognito:groups"];
 
-      return { isValidated: true, groups: groups };
+      return { isValidated: true, groups: groups, sub: userPayload.sub };
     } catch (e) {
```

Comment:
It would probably be good to come up with some kind of frontend supported error response that is sent that we can display in a popup or something as right now I have found that when I just throw errors it kills the program entirely.

Reply:
This isnt a problem here but would just be good in the future to have some way to standardize warnings to display for the user or something if they are errors the user should see.